app/scenes/Settings/People.js

```diff
@@ -87,6 +89,10 @@ class People extends React.Component<Props> {
           Invite people…
         </Button>
 
+        <TotalCount $loading={totalUsers === 0}>
```

Comment:
Looks like loading is managed within `Pagination`. This seems like a viable way to check since the totalUsers will always be at least 1

Reply:
Is using the dollar sign a pattern I might have missed with styled-components?

Reply:
dollar sign represent transient properties that only styled-components use, and will not transfer to the dom https://styled-components.com/docs/api#transient-props

Reply:
nice, TIL